src/server.js

```diff
@@ -116,70 +77,36 @@ function getScoreAndEvidence( skill, doc ) {
   return data;
 }
 
-// set the view engine to ejs
-app.set('view engine', 'ejs');
-
-// use res.render to load up an ejs view file
-
-// index page
-app.get('/', function(req, res) {
-  console.log( 'GET /' );
-  res.redirect( '/go' );
-});
-
-app.get('/go', function(req, res) {
-  console.log( 'GET /go' );
-  res.render( 'go' );
-});
-
-app.post('/go', async function (req, res) {
-  console.log(`POST /go`);
-
-  const docId = req.body.docid;
-  if (docId == '') {
-    res.redirect('/go');
-  } else {
-    let doc = await loadDoc(docId);
-    let role = doc.role;
-      role = roleNames[0];
-      doc.role = role;
-      await saveDoc(docId, doc);
-    }
-    res.redirect(`${hostname}/admin/${lcns(role)}/${docId}`);
-});
-
-// about page
-app.get('/about', function(req, res) {
-  console.log( `GET /about` );
-  res.render('pages/about');
-});
-
-function isValidRole( role ) {
+async function isValidRole(role) {
   let success = false;
-  let json = db[ role ]
+  if (!db) {
+    db = await rolesDb;
+  }
```

Comment:
Will the data ever mutate? If so this will mean once the application is running these values will be fixed